### ISSUE TITLE
The chat button on pulumi.com on 1/2 screen covers the cookie "Got it" button

### ISSUE THREAD

**Issue by theneva** [pulumi/docs#2315]
Hi! :wave:

I'm unfortunate enough to prefer browsing the web with Chrome taking up 1/2 of a MacBook Pro 15" screen at 100 % zoom. This lets me find a surprising number of visual bugs all over the web :smile:

### Problem description

The chat button covers the cookie "Got it" button on all of pulumi.com when browsing the page on exactly half of a MacBook Pro 15" screen width (latest Chrome, 100 % zoom, window height doesn't matter):

![image](https://user-images.githubusercontent.com/1404650/73257594-585b5f00-41c4-11ea-9c82-1b12e74511bb.png)

The cookie consent/confirmation button is not clickable at all without resizing the window, even though it clearly peeks out behind the chat button. I believe this is because the chat widget is pretty wide:

![image](https://user-images.githubusercontent.com/1404650/73257931-f18a7580-41c4-11ea-8825-a82ac5dbef8e.png)

### Affected product version(s)

The version of the website deployed at the time of writing.

### Suggestions for a fix

Dunno, to be honest 🤷‍♂ Make the chat widget only take up the width it needs, and/or push the "Got it" button slightly left, maybe?

**Comment by lukehoban** [comment]
Interestingly, this is only triggered in a fairly small range of viewport widths.  

![2020-11-13 22-06-38 2020-11-13 22_12_24](https://user-images.githubusercontent.com/223467/99141171-65372b00-25fd-11eb-8382-b6afafda18c7.gif)

**Comment by theneva** [comment]
Yep! Unfortunately, "Chrome filling half a MacBook Pro screen" falls inside that range - the web in general is full of, uh, interesting behaviour on that width.

Note that this exact width is the only one I care about! I don't mind if it's behaving strangely on any widths close to it. I know supporting every single possible width is incredibly difficult. I just want things to work when I snap Chrome to exactly half the screen 🙂
